src/app/locales/vi_VN.ts

```diff
@@ -0,0 +1,513 @@
+  },
+  // 'tsconfig-layout': {
+  //   action:
+  //     'Angular recomienda utilizar `tsconfig.base.json` para ayudar a organizar el tipado en diferentes contextos(en común, pruebas unitarias, pruebas end to end, aplicación, etc) `ng update` lo migrara automáticamente.',
+  // },
+  'classes-without-decorators': {
+    action:
+      'Ở phiên bản 10, các classes mà dùng đến các tính năng của Angular mà không có đi kèm 1 Angular Decorator nào sẽ không còn được hỗ trợ nữa. [Đọc thêm](https://v10.angular.io/guide/migration-undecorated-classes). `ng update` sẽ thực hiện việc thay đổi này cho bạn.'
+  },
+  'injectable-definitions': {
+    action:
+      'Từ phiên bản 9, việc sử dụng `@Injectable` cho hệ thống Dependency Injection đã trở nên nghiêm ngặt hơn và nếu như không khai báo hoàn chỉnh, sẽ xảy ra những hành vi không mong muốn. [Đọc thêm](https://v9.angular.io/guide/migration-injectable). `ng update` sẽ thực hiện việc thay đổi này cho bạn.'
+  },
+  'closure-jsdoc-comments': {
+    action:
+      'Các thư viện của Angular không còn chứa những bình luận (comments) JSDOC nữa, mà những bình luận này cần thiết cho Closure Compiler (rất hiếm). Việc hỗ trợ này chỉ mang tính chất thử nghiệm và chỉ hoạt động trong một số trường hợp nhất định mà thôi. Chúng tôi sẽ thông báo về một cách khác được khuyến khích hơn trong thời gian ngắn sắp tới.'
+  },
+  'forms-number-input': {
+    action:
+      'Nếu bạn dùng Angular Forms, những trường nhập liệu (inputs) có kiểu là `number` sẽ không còn lắng nghe đến [các sự kiện thay đổi](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) nữa (những sự kiện này không nhất thiết sẽ phải sinh ra cho mỗi thay đổi của giá trị đầu vào), thay vào đó bạn nên lắng nghe [các sự kiện đầu vào (input events)](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event).'
+  },
+  'forms-length-input': {
+    action:
+      'Đối với việc kiểm tra sự hợp lệ của Angular Forms, các phép kiểm tra `minLength` và `maxLength` từ giờ sẽ kiểm tra và sẽ chỉ kiểm tra độ dài (length) của giá trị của Form Control nếu như giá trị đó có thuộc tính `length`.'
+  },
+  'esm5-bundles': {
+    action:
+      '[Angular Package Format](https://g.co/ng/apf) đã được cập nhật để xoá bỏ định dạng `esm5` và `fesm5`. Những định dạng này sẽ không được phân phối ở bên trong những thư viện NPM của chúng tôi nữa. Nếu như bạn không dùng Angular CLI, có thể bạn sẽ cần phải tự mình xuống cấp (downlevel) Angular code xuống ES5.'
+  },
+  'console-errors': {
+    action:
+      'Những cảnh báo về nguyên tố không nguồn gốc (Unknown Elements) từ giờ sẽ được ghi ra Console dưới dạng Lỗi. Điều này sẽ không làm hư ứng dụng của bạn, nhưng có thể sẽ làm ảnh hưởng đến một số công cụ mà những công cụ này sẽ không hoạt động nếu như có thông tin được ghi ra Console dưới dạng Lỗi (`console.error`)'
+  },
```

Comment:
`nguyên tố ` sounds very chemistry to me 😂  We could keep it as `Element`.

Reply:
I'll think about it. It does sound weird to me.

Reply:
changed to: thành phần